### DISCUSSION TITLE
[Feature] Expand/Collapse archived pics of a person

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6009] [Feature Request]
### The feature

Currently:
If you start archiving pictures, they are not shown anymore in the People albums. So one person of the past may have "0" pic associated to it if all the pics are actually archived.

Suggestion:
Add a toggle switch that allows expanding the archived pics of this person, and collapsing it.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Zhigang26** [comment]
This is also what I hope to have! Now when I create albums, I will tag the pictures and videos also and then archive them. Recently the archive also provide the timeline, it's great. But the assets archived, it will not shown in the people.

As @diyoyo mentioned above, add a toggle switch or two tags (Archived, not Archived) to allow expanding the archived pics of this people, and collapsing it, that will be great.

Thanks.

**Comment by Zhigang26** [comment]
Will this be considered?

**Reply by bo0tzz** [reply]
If a feature request is still open that means it's up for consideration. No need to bump it.

**Reply by Zhigang26** [reply]
Sorry to bother you, still want to bump this feature request after half year. 😂
